lib/versioning/poetry/index.ts

```diff
@@ -130,6 +131,20 @@ function getNewValue({
 }: NewValueConfig): string {
   if (rangeStrategy === 'replace') {
     const npmCurrentValue = poetry2npm(currentValue);
+    try {
+      const massagedNewVersion = padZeroes(newVersion);
+      if (
+        npm.isVersion(massagedNewVersion) &&
+        npm.matches(massagedNewVersion, npmCurrentValue)
+      ) {
+        return currentValue;
+      }
+    } catch (err) /* istanbul ignore next */ {
+      logger.info(
+        { err },
+        'Poetry versioning: Error caught checking if newVersion satisfies currentValue'
+      );
```

Comment:
When will this happen? Shouldn't be better have a test?

Reply:
I'm not aware of any case that can trigger it, so it's there purely defensively. I don't want to make it a WARN in case it happens a lot.. in that case I'll check the app logs once deployed for any logs of this and try to fix.